docs/datasets/stack.ipynb

```diff
@@ -0,0 +1,362 @@
+{
```

Comment:
<p>Maybe add that if all the dataset is masked, exposure and livetime is not increased. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>